### PR TITLE
Add ActiveMQ configuration

### PR DESCRIPTION
**Pull request by fabricetriboix**

### CODE REVIEW

**Comment by david-oc-miller** [comment]
@fabricetriboix merge conflicts here  :-(

**Comment by fabricetriboix** [comment]
> @fabricetriboix merge conflicts here :-(

OK, I'll have a look

File: CloudFormation/amazonmq.yml [hunk 1]
```diff
@@ -224,12 +224,215 @@ Resources:
...
+              </plugins>
+              <!--
+              Network connectors let you connect brokers into networks of brokers.
+              For more information, see Creating and Configuring an Amazon MQ Network of Brokers
+              (https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/amazon-mq-creating-configuring-network-of-brokers.html)
+              in the Amazon MQ Developer Guide and also Networks of Brokers
+              (http://activemq.apache.org/networks-of-brokers.html) in the ActiveMQ documentation.
+              -->
+              <!--
+              <networkConnectors>
```
Comment (david-oc-miller) [inline CloudFormation/amazonmq.yml:417]: it looks like this is the standard default configuration, and these elements are all commented out, and basically exist for documentation purposes, right?

Comment (fabricetriboix) [reply CloudFormation/amazonmq.yml:417]: Yes, I copied the standard ActiveMQ configuration given by AWS

**Pull request merged**